public/diagnostic.js

```diff
@@ -23,3 +23,29 @@
 //  -  Order 1 : 2 hats ($5 each)
 //  -  Order 2 : 1 desk lamp ($20 each)
 //  -  Order 3 : 3 hand towels ($8 each)
+
+const Cart = Ember.Object.extend({
+  addToCart: [],//Ember.thingy(),
+  totalPrice: []//Math.sum(orderPrice);
```

Comment:
`totalPrice` should be a computed property, and computed properties should be set in the Ember Class.